### PR TITLE
Assets expiration changes

### PR DESCRIPTION
**Pull request by vytis**
To improve handling of assets for team users we need to change the lifetime to eternal.

### CODE REVIEW

File: Source/Request Strategies/Assets/Helpers/AssetRequestFactory+Retention.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+extension AssetRequestFactory {
+    static func defaultAssetRetention(for sender: ZMUser, in conversation: ZMConversation) -> Retention {
+        if sender.isTeamMember || sender.team != nil {
```
Comment (daehn) [inline Source/Request Strategies/Assets/Helpers/AssetRequestFactory+Retention.swift]: The `conversation` parameter seems not to be used in the method body.

Comment (vytis) [reply Source/Request Strategies/Assets/Helpers/AssetRequestFactory+Retention.swift]: good catch, it should be conversation, not sender!

Comment (vytis) [reply Source/Request Strategies/Assets/Helpers/AssetRequestFactory+Retention.swift]: I should as well add tests for this method at least

**Comment by vytis** [comment]
@daehn fixed the problem and added tests

**Pull request merged**